src/recensio/plone/adapter/indexer.py

```diff
@@ -0,0 +1,18 @@
+from plone.indexer.decorator import indexer
+from recensio.plone.interfaces import IReview
+
+
+@indexer(IReview)
+def authorsUID(obj):
+    authors = (
+        getattr(obj, "authors", [])
+        + getattr(obj, "reviewAuthors", [])
+        + getattr(obj, "editorial", [])
+        + getattr(obj, "curators", [])
```

Comment:
It might be that something like `getattr(obj, "field", []) or []` is needed if the default is `None`.
Anyway we will find out if that happens...